models/users.js

```diff
@@ -572,6 +572,41 @@ const fetchAllUsers = async () => {
   return users;
 };
 
+const archiveUserIfNotInDiscord = async () => {
+  try {
+    const snapshot = await userModel.where("roles.in_discord", "==", false).get();
+    const batch = firestore.batch();
+    const summary = {
+      totalUsersWithArchivedRoleUpdated: 0,
```

Comment:
`totalUsersWithArchivedRoleUpdated` is too verbose. can we call it `totalArchivedUsersUpdated`